spp_custom/__manifest__.py

```diff
@@ -16,6 +16,7 @@
     "data": [
         "data/product_sequence.xml",
         "views/product.xml",
+        "views/product_supplierinfo_vendor_code_view.xml",
```

Comment:
For file naming, we try to follow the [oca conventions](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming). That would be here: `views/product_supplierinfo_views.xml`.

\* to be fair, we mostly name according to `views/<model>.xml` but we're _tending_ towards oca conventions ;-)

Reply:
Excellent, I was indeed struggling to decide how to name the file, so thank you for pointing out the reference. :)